### PR TITLE
Add 'onCompleteThrows', 'onErrorThrows' test for CallbackCompletable

### PR DESCRIPTION
**Pull request by ggikko**
This is test code for CallbackCompletable.

### CODE REVIEW

File: src/test/java/io/reactivex/internal/observers/CallbackCompletableObserverTest.java [hunk 1]
```diff
@@ -35,4 +41,73 @@ public void customOnErrorShouldReportCustomOnError() {
...
+
+        assertTrue(o.isDisposed());
+
+        RxJavaPlugins.reset();
+    }
+
+    @Test
+    public void onErrorCThrows() {
+
+        List<Throwable> errors = TestHelper.trackPluginErrors();
```
Comment (akarnokd) [inline src/test/java/io/reactivex/internal/observers/CallbackCompletableObserverTest.java]: Code between this and the `RxJavaPlugins.reset()` should be put in try-finally.

**Review by akarnokd — COMMENTED** [review]
The class itself is definitely covered to the max already: https://codecov.io/gh/ReactiveX/RxJava/src/2.x/src/main/java/io/reactivex/internal/observers/CallbackCompletableObserver.java

**Comment by ggikko** [comment]
The class itself is definitely covered to the max already:  

Is this means that pr is not needed.. 

I didn't know how much code is covered by test code. 

I always appreciate your comment. thanks.

**Comment by akarnokd** [comment]
No problem. The [coverage](https://github.com/ReactiveX/RxJava/pull/5627#issuecomment-333061122) reports usually list classes with less than 100% coverage. You can attempt to improve a few of those but note that sometimes it is impossible to cover everything due to hidden bytecodes generated by the javac that can never actually run.

**Pull request closed without merging**